torch/csrc/jit/register_c10_ops.cpp

```diff
@@ -7,13 +7,64 @@ namespace torch {
 namespace jit {
 namespace {
 
-at::Tensor unwrap(at::Tensor&& tensor) {
+at::Tensor unwrap_tensor(at::Tensor&& tensor) {
   if (tensor.requires_grad()) {
     throw std::runtime_error("Autograd not yet supported for c10 ops.");
   }
   return torch::autograd::Variable(std::move(tensor)).data();
 }
 
+IValue unwrap(IValue&& ivalue) {
```

Comment:
I know you're not changing any behavior for Tensor here, but will this work for non variables?

Reply:
no, this only works with variables. But afaik, jit ops are always called with variables.